### PR TITLE
 Quality check: Metadata updates in AAC Content Pattern Library

### PR DESCRIPTION
**Pull request by v-stsavell**
This PR updates metadata in the AAC templates. Here's the [ADO work item](https://dev.azure.com/msft-skilling/Content/_workitems/edit/314493).

### CODE REVIEW

File: templates/org/aac/aac-topic-landing-page.md [hunk 1]
```diff
@@ -3,6 +3,8 @@ title: TOPIC NAME on Azure
 description: A guide to building running TOPIC NAME on Azure
 author: 
 ms.date: 2/4/2019
+ms.topic: landing-page
+ms.service: azure-architecture-center
 ---
```
Comment (ShannonLeavitt) [inline templates/org/aac/aac-topic-landing-page.md:9]:
```suggestion
ms.subservice: 
---
```
Adding an empty slug here

File: templates/org/aac/aac-topic-landing-page.md [hunk 2]
```diff
@@ -3,6 +3,8 @@ title: TOPIC NAME on Azure
 description: A guide to building running TOPIC NAME on Azure
 author:
```
Comment (ShannonLeavitt) [inline templates/org/aac/aac-topic-landing-page.md]:
```suggestion
author: <Contributor's GitHub username>
ms.author: <Contributor's Microsoft alias> 
```

**Comment by v-stsavell** [comment]
@ckittel - requesting review. Thanks!

File: templates/org/aac/aac-topic-landing-page.md [hunk 3]
```diff
@@ -1,8 +1,12 @@
 ---
 title: TOPIC NAME on Azure
 description: A guide to building running TOPIC NAME on Azure
-author: 
+author: <Contributor's GitHub username>
+ms.author: <Contributor's Microsoft alias> 
 ms.date: 2/4/2019
+ms.topic: landing-page
+ms.service: azure-architecture-center
+ms.subservice:
```
Comment (ckittel) [inline templates/org/aac/aac-topic-landing-page.md]:
```suggestion
```

File: templates/org/aac/aac-browser-header.yml [hunk 4]
```diff
@@ -2,13 +2,13 @@
...
-  ms.author: <Contributor's Microsoft alias. Can include multiple contributors, separated by commas. If there's no alias, use the Microsoft email alias "architectures".>
-  ms.date: <Publish or major update date - mm/dd/yyyy>
-  ms.topic: conceptual
-  ms.service: architecture-center
-  ms.subservice: example-scenario
+  description: <Write a 100-160 character description that starts with a call to action and ends with a period. This becomes the browse card description.>
+  author: <Contributor's GitHub username>
+  ms.author: <Contributor's Microsoft alias>
+  ms.date: <Date of publish or last freshness pass, in mm/dd/yyyy format>
+  ms.topic: <For an example workload, use `example-scenario`; for a reference architecture, use `reference-architecture`; for a solution idea, use `solution-idea`.>
```
Comment (ckittel) [inline templates/org/aac/aac-browser-header.yml]:
```suggestion
  ms.topic: <For an example workload, use example-scenario; for a reference architecture, use reference-architecture; for a solution idea, use solution-idea>
```

File: templates/org/aac/aac-browser-header.yml [hunk 5]
```diff
@@ -2,13 +2,13 @@
...
+  ms.service: azure-architecture-center
+  ms.subservice: <For an example workload, use `example-scenario`; for a reference architecture, use `reference-architecture`; for a solution idea, use `solution-idea`.>
```
Comment (ckittel) [inline templates/org/aac/aac-browser-header.yml]:
```suggestion
  ms.subservice: <For an example workload, use example-scenario; for a reference architecture, use reference-architecture; for a solution idea, use solution-idea>
```

**Review by ckittel — CHANGES_REQUESTED** [review]
I don't know what https://github.com/v-stsavell/content-templates/blob/metadata-update/templates/org/aac/aac-architecture-guide-browse.yml is -- I don't THINK it's used anywhere, but just for completeness, can you please change `ms.service` to `azure-architecture-center` and `ms.subservice` to `architecture-guide`  in that file - You can make the `ms.date` and author changes as well if you wish.

**Review by ckittel — APPROVED** [review]
Thanks!

**Comment by ckittel** [comment]
#sign-off (though, that doesn't work in this repo -- it's the thought that counts).  Make sure you @ whoever has permissions in this repo to merge.  I don't remember who, maybe check old PRs.

**Comment by v-stsavell** [comment]
Hi @mattbriggs - I see you've previously merged PRs in this repo. Could you merge, or direct me to who can? Thanks!

**Review by mattbriggs — APPROVED** [review]

**Pull request merged**